src/sybil/table_query.go

```diff
@@ -311,7 +316,7 @@ func (t *Table) LoadAndQueryRecords(loadSpec *LoadSpec, querySpec *QuerySpec) in
 		mu.Unlock()
 		count += rowStoreQuery.count
 
-		if !DELETE_BLOCKS_AFTER_QUERY && t.RowBlock != nil {
+		if loadSpec != nil && loadSpec.SkipDeleteBlocksAfterQuery && t.RowBlock != nil {
```

Comment:
how is loadSpec nil again?

Reply:
in the current codebase it won't be. I generally prefer making called code avoid panics but I don't have a strong preference.